crates/oxide/src/glob.rs

```diff
@@ -60,7 +64,11 @@ fn get_fast_patterns(base_path: &Path, patterns: &Vec<String>) -> Vec<(PathBuf,
             // Safety: We know that the length is greater than 1, so we can safely unwrap.
             let file_pattern = folders.pop().unwrap();
             let all_folders = folders.clone();
-            let mut temp_paths = vec![base_path.to_path_buf()];
+            let mut temp_paths = if is_absolute_pattern {
+                vec![PathBuf::from("/")]
+            } else {
+                vec![base_path.to_path_buf()]
+            };
```

Comment:
Let's update this to assume we are always getting relative paths because handling absolute paths is a bit more complicated across different operating systems and I'd rather handle that as a separate scope of work 👍